usaspending_api/tests/test_award_index_elasticsearch.py

```diff
@@ -102,11 +102,6 @@ def test_date_range(award_data_fixture, elasticsearch_award_index):
 
 def test_tas(award_data_fixture, elasticsearch_award_index):
     elasticsearch_award_index.update_index()
-    search_regex = (
```

Comment:
Is this regex in line with the actual code implementation?

Reply:
It is not, which is why it was deleted. The new regex is on line 115, which still doesn't match the code, but to my understanding this is testing the index, not the filter. If we don't think it's important to  test the index by itself, I would rather just delete this test suite and trust the filter tests to test filters.